MbDotNet/MountebankRequestProxy.cs

```diff
@@ -121,6 +123,24 @@ private HttpResponseMessage ExecutePost(string resource, string json)
             }
         }
 
+        private void HandleDynamicPort(HttpResponseMessage response, Imposter imposter)
+        {
+            if (imposter.Port == null)
+            {
+                try
+                {
+                    var content = response.Content.ReadAsStringAsync().Result;
+                    Match match = new Regex(".*\"port\": (\\d+),", RegexOptions.Multiline).Match(content);
+                    imposter.SetDynamicPort(Convert.ToInt32(match.Groups[1].Value));

```

Comment:
I think we should use `int.Parse` here instead of `Convert.ToInt32`. That way it will throw an exception instead of silently setting the port to zero.